docker/app.entrypoint.sh

```diff
@@ -39,7 +39,10 @@ if ! $( wp core is-installed --allow-root ); then
 		--allow-root
 fi
 
-# Install and activate WPGatsby
+echo "Running WordPress version: $(wp core version --allow-root)"
+
+# Install and activate wp-graphql
+wp plugin install wp-graphql --allow-root
```

Comment:
Will this install WPGraphQL from the .org repo? 

Don't we want to use the current version of the repo and _not_ install it from .org repo? 🤔

Reply:
This was to put a version of wp-graphql in the running docker when the plugin is not already mounted in the running docker image.  It'll  use what is mounted locally in your working directory.  So in most cases it'll just say "already installed".  
I might be getting ahead of myself or out of order for the followup work where we use this docker image with another wp-graphql plugin.  In that case, wp-graphql is not found locally and is not installed, so my initial thought was it needs to be installed. I haven't fully flushed that work out.

Reply:
Maybe we move something like we do in acf when a variable Is specified https://github.com/wp-graphql/wp-graphql-acf/blob/develop/docker/app.entrypoint.sh#L47

Reply:
Removing that for now until I have a better understanding and use case.